src/utilities/auth.js

```diff
@@ -0,0 +1,38 @@
+import store from '../redux/store';
+import {paths, roles} from './constants';
+
+const { ADMIN, SUPERVISOR, EDITOR } = roles;
```

Comment:
We have also coordinator role 

https://github.com/technologiesforaccessibility/osmintegrator-wiki/wiki/Permissions-and-Roles#coordinator